app/helpers/application_helper.rb

```diff
@@ -4,32 +4,33 @@ module ApplicationHelper
 
   # Add autogenerated html data-id attribute if not present (prefixed with "aid_")
   def link_to(*args, &block)
```

Comment:
Could you remove `&block`? It's supposed to be faster, https://github.com/JuanitoFatas/fast-ruby/blob/master/code/proc-and-block/proc-call-vs-yield.rb

Reply:
done